src/app/components/error/ErrorBoundary.js

```diff
@@ -23,10 +31,24 @@ class MainErrorBoundary extends React.Component {
   }
 
   componentDidCatch(error, errorInfo) {
+    const { component, intl } = this.props;
+
     if (this.airbrake) {
+      const { dbid, email, name } = window.Check.store.getState().app.context.currentUser;
       this.airbrake.notify({
-        error,
+        // For some reason we need to reinstantiate the error or pass the error as string.
+        // Not doing this causes the additional info (contex, params, session) to be empty
+        error: new Error(error),
+        context: { component },
         params: { info: errorInfo },
+        session: { name, dbid, email },
+      }).then((response) => {
+        if (Intercom) {
```

Comment:
Didn't you need to declare "global Intercom" on top? The Linter didn't complain?